scripts/delayed.js

```diff
@@ -23,7 +23,9 @@ function injectScript(src) {
 
 function decorateTwitterFeed() {
   const anchors = main.getElementsByTagName('a');
-  const twitterAnchors = Array.from(anchors).filter((a) => a.href.includes('twitter') && a.href.includes('ref_src'));
+  const twitterAnchors = Array.from(anchors).filter(
+    (a) => a.href.includes('twitter') && a.href.includes('ref_src'),
```

Comment:
is this really a save way to identify twitter links? shouldn't we rather check for the domain?

Reply:
I tend to agree... this would also match `https://www.facebook.com/twitter` ;)